ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/qos-profiles.j2

```diff
@@ -22,34 +22,10 @@ qos profile {{ profile.name }}
    service-policy type qos input {{ profile.service_policy.type.qos_input }}
 {%     endif %}
 {%     for tx_queue in profile.tx_queues | arista.avd.natural_sort('id') %}
-   !
-   tx-queue {{ tx_queue.id }}
-{%         if tx_queue.bandwidth_percent is arista.avd.defined %}
-      bandwidth percent {{ tx_queue.bandwidth_percent }}
-{%         elif tx_queue.bandwidth_guaranteed_percent is arista.avd.defined %}
-      bandwidth guaranteed percent {{ tx_queue.bandwidth_guaranteed_percent }}
-{%         endif %}
-{%         if tx_queue.priority is arista.avd.defined %}
-      {{ tx_queue.priority }}
-{%         endif %}
-{%         if tx_queue.shape.rate is arista.avd.defined %}
-      shape rate {{ tx_queue.shape.rate }}
-{%         endif %}
+{%         include 'eos/tx-queues.j2' %}
```

Comment:
Adding these extra sub-sub templates to enable a bit of reuse will make it more complex to understand the templates later. @carlbuchmann @gmuloc where do we set the bar here? All the other features under the qos profiles could also be shared, and same for many other areas of eos_cli_config_gen.
IMO we should stay conservative on this.

No matter what we need to clearly name the files so we can identify the hierarchy. Maybe these should be `ethernet-interface-tx-queues.j2`

Reply:
Happy to rename these, but wouldn't that mean that in the `qos-profiles.j2` we would be making reference to the `ethernet-interface-tx-queues.j2` ? I named it `tx-queues.j2` to make it independent of whether it was in the profile or the ethernet level.

Reply:
I understand your concern on naming, but I would rather have qos-profiles use the ethernet-interface... variant than confusing "tx-queues" with some global key. In general I would prefer not to have this reuse at all, but I understand why you do it.

Reply:
Understood, renames these templates in the latest commit.